pyro/infer/mcmc/hmc.py

```diff
@@ -411,6 +411,6 @@ def sample(self, trace):
 
     def diagnostics(self):
         return OrderedDict([
-            ("Step size", self.step_size),
-            ("Acceptance rate", self._accept_cnt / self._t)
+            ("step size", "{:.2e}".format(self.step_size)),
+            ("acc. rate", "{:.3f}".format(self._accept_cnt / self._t))
```

Comment:
I have wanted this for a while. Your trick `"{:.2e}".format(...)` is +1000 to me.